examples/rust/devenv.yaml

```diff
@@ -4,3 +4,5 @@ inputs:
     inputs:
       nixpkgs:
         follows: nixpkgs
+  devenv:
```

Comment:
this needs to be removed, you can use `devenv-run-tests` and it will automatically do the overriding